README.md

```diff
@@ -167,5 +197,16 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+
+lamp = Lamp.create(wishes: 3)
+genie = Genie.create(name:"Genie")
+lamp.update(genie: genie)
+update.lamp(wishes: 1)
+jafar = Genie.create(name: "Jafar")
+jafarsLamp = newLamp.create(wishes: 3)
```

Comment:
`.create` is a class method. It can only be called on a class (e.g., `Lamp.create( )`).